frontend/utils/data.ts

```diff
@@ -0,0 +1,65 @@
+// api/profiles
+[
+    {
+        "id": 10,
+        "url": "http://chitchat-api.vndprojects.com/api/profiles/10/",
+        "user": "testjane",
+        "follows": [
+            {
+                "username": "testjane",
+                "url": "http://chitchat-api.vndprojects.com/api/profiles/10/"
+            }
+        ],
+        "followers": [
+            {
+                "username": "testjane",
+                "url": "http://chitchat-api.vndprojects.com/api/profiles/10/"
+            }
+        ],
+        "profile_picture": null,
+        "first_name": "",
+        "last_name": ""
+    },
+    
+    {
+        "id": 10,
+        "url": "http://chitchat-api.vndprojects.com/api/profiles/10/",
+        "user": "testjane",
+        "follows": [
+            {
+                "username": "testjane",
+                "url": "http://chitchat-api.vndprojects.com/api/profiles/10/"
+            }
+        ],
+        "followers": [
+            {
+                "username": "testjane",
+                "url": "http://chitchat-api.vndprojects.com/api/profiles/10/"
+            }
+        ],
+        "profile_picture": null,
+        "first_name": "",
+        "last_name": ""
+    }
+];
+
+// api/posts
+
+[
+    {
+        "id": 1,
+        "url": "http://chitchat-api.vndprojects.com/api/posts/1/",
+        "user_profile": "http://chitchat-api.vndprojects.com/api/profiles/7/",
+        "body": "Test post",
+        "created_at": "2024-02-08T15:03:56.628379Z",
+        "likes_count": 1
+    },
+    {
+        "id": 2,
+        "url": "http://chitchat-api.vndprojects.com/api/posts/2/",
+        "user_profile": "http://chitchat-api.vndprojects.com/api/profiles/4/",
+        "body": "Another one",
+        "created_at": "2024-02-08T15:17:16.231555Z",
+        "likes_count": 0
+    },
+]
```

Comment:
@Presson-coder, is this for testing purposes?

Reply:
ohh that file! I was just using it to be seeing the nature of the endpoints and responses locally but otherwise is of no use ,could have deleted it after I was done.

Reply:
@Presson-coder No problem, I understand. If you're using it for local testing, let's remove it from git while keeping it in your filesystem so you can continue to test against it. Do:

```shell
git rm --cached frontend/utils/data.ts
```

Reply:
Ok sure thank you